lib/extensions/extensions.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
```

Comment:
Refactoring needed: move `lib/extensions` to `lib/qar/extensions`

Reply:
fixed in 3e6cdfd